posts/september-2025-restructure.md

```diff
@@ -0,0 +1,90 @@
+
+I think it's fair to say that at this stage I've lost faith in the nonprofit buisness model working in a way that allows Exercism to reach any of its potential.
+Keeping something free for everyone relies on either the user being the product, or on significant donations, and without either, it's very hard to grow.
+
+## Erik + OSS Restructure
+
+Probably the hardest thing about the situation right now, is that we can no longer afford to pay Erik, so he's leaving as an employee at the end of this week.
+
+Erik has been an absolute critical part of Exercism's growth and success over the last few years.
+He's also been a wonderful colleague and friend, and I'll really miss working alongside him.
+It goes without saying that I'm incredbily grateful to Erik for all his hard work and support.
+And I know many of you will feel the same (if so, please reach out and tell him!)
+
+Erik's a die-hard Exercism fan, and he's going to continue as a senior maintainer of a few tracks, and he'll hold onto his super-admin priviledges, but the plethera of hidden (and often a bit boring) things that he does day-to-day need to get spread across the organisation.
+
+The one key thing we're enforcing is that every PR in a live Exercism repository should get a review before it can be merged (with only one exception outlined below). 
+This has generally been the case anyway for a long time, but there are places where it's fallen through the cracks, so we've now scripted things to ensure this is always the case.
+
+To do this, we've come up with some new classifications system for repos, and specific rules for each type:
+- `maintained`: A repo that has multiple maintainers. All PRs require reviews from a track maintainer.
+- `maintained-solitary`: A repo that has one maintainer. A new cross-track-maintainers team will get pinged to review all PRs.
+- `unmaintained`: A repo that has no maintainers. The cross-track-maintainers team will get pinged here too.
+- `maintained-autonomous`: A repo where all maintainers are also in the cross-track-maintainer team. This is the exception, where they can merge their own PRs.
+- `wip-track`: An unlaunched track. As it's not "live" yet, it doesn't have restrictions.
+
+We've created two new GitHub teams that enforce this.
+- `@exercism/guardians`: A team to check the safety of PRs to tooling repositories (test runners, analyzers, etc). Made up of a few longstanding polyglots.
+- `@exercism/cross-track-maintainers`: A new team made up of longstanding polyglots who are active on the site on a day-to-day basis, and who have the same level of reviewing-pedanticism that I do (ie they're not more strict or more flexible than me). This is important, as I want a consistent, responsive experience from this team.
+
+Both teams are invite-only.
+I'll review them sporadically.
+
+We've also "pruned" a lot of inactive maintainers as part of this. 
```

Comment:
We've not really done this yet though, only for Python and Go. But we now have the tools to be able to do this.

Reply:
Let's do it before we push the blog post out.